### ISSUE TITLE
Any way to sync whole model with the view at one go?

### ISSUE THREAD

**Issue by shrutizamwar** [NYTimes/backbone.stickit#223]
Hi,

I have been using stickit and everything seems perfect!
I just had one doubt. Is there any way I can change the view without updating the model and change the model after all the elements in the view are changed? Like for e.g. I have a form with different input elements and a submit button. So here I keep changing the view elements. but the model is not updated as of yet. Now when I click on submit button, I call this API on view say "this.sync()" which now updates the whole model with the view.

Basically I don't want the model to be updated with the immediate change in the view. I want model to be in sync with the view when the user performs some action.
